irony-cdb.el

```diff
@@ -63,6 +63,20 @@ for files that it cannot handle."
   :type '(repeat function)
   :group 'irony-cdb)
 
+(defcustom irony-cdb-search-directory-list '("." "build")
```

Comment:
Why "." is needed?

I would remove "build" from the default, I'm not convinced about its general adoption. Also I prefer these directories to be opt-in, removing a directory is often difficult.